### DISCUSSION TITLE
ToolJet Secures Funding from Microsoft's M12 GitHub Fund 🚀

### DISCUSSION THREAD

**Discussion by Navaneeth-pk** [ToolJet/ToolJet#7123] [Announcements]
![image](https://github.com/ToolJet/ToolJet/assets/7828962/84951ea2-f857-4df5-b6bf-d5eff8a2849d)

Two years ago, we [open-sourced](https://news.ycombinator.com/item?id=27421408) ToolJet on GitHub. Today, we are excited to announce that ToolJet has raised a funding round from [Microsoft's M12 GitHub Fund](https://m12.vc/news/announcing-the-m12-github-fund/?ref=blog.tooljet.com). This is a huge milestone for us, and we're grateful for the support of these two amazing companies.

We believe that low-code development is a key way for businesses to accelerate their innovation. Low-code platforms make it possible for businesses to build applications quickly and easily, without having to rely on expensive and time-consuming custom development.

Microsoft and GitHub are both leaders in the tech industry, and their investment will help us to accelerate our growth and reach more customers. We're also excited to collaborate with them on new products and features that will make ToolJet even more powerful. The partnership will help us expand our team, develop new features, and grow our community. It will also help us integrate AI and machine learning capabilities into the platform. This will allow ToolJet users to build more intelligent and custom applications.

### What's next?

We are excited about the future of ToolJet. With this investment, we are well-positioned to accelerate our growth and reach more businesses around the world. We believe that low-code development is the future of software development, and we are committed to making ToolJet the leading low-code platform in the world.

We will be sharing more updates about our plans for the future in the coming months.

We also want to thank the ToolJet community for your support. Your feedback and suggestions have helped us to build a better platform. We're committed to continuing to work with you to make ToolJet the best it can be.

Every star helps us to increase our visibility and reach. This means that more people will be able to find and use ToolJet, which will help us to achieve our mission of making building internal tools easier for everyone 🌟.

**Comment by Waliyudini** [comment]
Wow congratulations 🎉
